openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -240,21 +240,61 @@ <h2 class="staff-info__title ui-staff__subtitle ui-toggle-visibility__control">
             </h2>
             <div class="ui-toggle-visibility__content">
                 {% if workflow_status == "done" %}
-                    <p>
+                    <p class="staff-info__final__grade__score">
                     {% with points_earned_string=score.points_earned|stringformat:"s" points_possible_string=score.points_possible|stringformat:"s" %}
                         {% blocktrans with points_earned='<span class="grade__value__earned">'|safe|add:points_earned_string|add:'</span>'|safe points_possible='<span class="grade__value__potential">'|safe|add:points_possible_string|add:'</span>'|safe %}
                             Final grade: {{ points_earned }} out of {{ points_possible }}
                         {% endblocktrans %}
                     {% endwith %}
                     </p>
+                    <table class="staff-info__status__table staff-info__final__grade__table" summary="{% trans "Final Grade Details" %}">
+                        <thead>
+                        <tr>
+                            <th abbr="{% trans 'Criterion' %}" scope="col">{% trans "Criterion" %}</th>
+                            {% with criterion=grade_details.criteria.0 %}
+                                {% for assessment in criterion.assessments %}
+                                    <th abbr="{{ assessment.title }}" scope="col">{{ assessment.title }}</th>
+                                {% endfor %}
+                            {% endwith %}
+                        </tr>
+                        </thead>
+
+                        <tbody>
+                        {% for criterion in grade_details.criteria %}
+                            <tr>
+                                <td class="label">{{ criterion.label }}</td>
+                                {% for assessment in criterion.assessments %}
+                                    <td class="value">
+                                    {% if assessment.points != None %}
+                                        <div>
+                                        {% blocktrans with assessment_label=assessment.option.label count points=assessment.points %}
+                                            {{ assessment_label }} - {{ points }} point
+                                        {% plural %}
+                                            {{ assessment_label }} - {{ points }} points
```

Comment:
Screencap of my sandbox with 0.2.6 translations being pushed:

<img width="930" alt="screen shot 2016-01-21 at 4 23 12 pm" src="https://cloud.githubusercontent.com/assets/7373924/12495033/5a7721f0-c05b-11e5-8ded-8bad319f1285.png">

Is this okay?I think translators will be able to translate the `blocktrans` above into RTL languages properly, so I'm good with this if everyone else is.